Formula/s/sevenzip.rb

```diff
@@ -25,27 +25,38 @@ class Sevenzip < Formula
   end
 
   def install
+    mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
+    mk_suffix, directory = if OS.mac?
+      ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
+    else
+      ["gcc", "g"]
+    end
     cd "CPP/7zip/Bundles/Alone2" do
-      mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
-      mk_suffix, directory = if OS.mac?
-        ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
-      else
-        ["gcc", "g"]
-      end
-
       system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
 
       # Cherry pick the binary manually. This should be changed to something
       # like `make install' if the upstream adds an install target.
       # See: https://sourceforge.net/p/sevenzip/discussion/45797/thread/1d5b04f2f1/
       bin.install "b/#{directory}/7zz"
     end
+    cd "CPP/7zip/Bundles/Format7zF" do
+      system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
+      if OS.mac?
+        lib.install "b/#{directory}/7z.so" => "7z.dylib"
+      else
+        lib.install "b/#{directory}/7z.so"
+      end
+    end
   end
 
   test do
     (testpath/"foo.txt").write("hello world!\n")
     system bin/"7zz", "a", "-t7z", "foo.7z", "foo.txt"
     system bin/"7zz", "e", "foo.7z", "-oout"
     assert_equal "hello world!\n", (testpath/"out/foo.txt").read
+
+    lib7z = OS.mac? ? lib/"7z.dylib" : lib/"7z.so"
+    assert_path_exists lib7z, "7-Zip shared library was not installed"
+    system "file", lib7z
```

Comment:
This doesn't really test that the library works. Can we test that it can be linked with and used?